app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Public Culinary Recipes</h1>
+
+<ul>
+  <% @culinary_recipes.each do |recipe| %>
+    <li>
+      <h2><%= recipe.name %></h2>
+      <p><%= recipe.description %></p>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- [ ] Great work so far! As per the project requirements for the public recipes index page, each recipe should display the recipe name, total food item, total price, and the owner's name. However, it appears that your current implementation does not match the design. I would recommend that you follow the design and update your code accordingly. 
- [ ] Additionally, the title should be in the top center. Please update your page title too.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/AlaaAlsalem/recipe/assets/55840999/3e045005-9b60-4a10-a1e2-e7ae34fe415b)| ![image](https://github.com/AlaaAlsalem/recipe/assets/55840999/05f7fd3c-0b4e-4f97-ba20-5bea405f60ea)|